test/test_integration_variable.py

```diff
@@ -0,0 +1,225 @@
+        value_map = {}
+        for variable in variables:
+            value = uuid.uuid4().hex
+            Utils.set_variable(self, variable, value)
+            value_map[variable] = value
+
+        batch_result_string = Utils.get_variable(self, *variables)
+        batch_result = json.loads(batch_result_string)
+
+        for variable_name, variable_value in value_map.items():
+            self.assertEquals(variable_value, batch_result[variable_name])
+
+        os.remove(file_name)
+
+    @integration_test
+    def test_batch_existing_and_nonexistent_variable_raises_error(self):
+        output = self.invoke_cli(self.cli_auth_params,
+                                 ['variable', 'get', '-i', 'one/password,unknown'], exit_code=1)
+        self.assertIn("404 Client Error", output)
+
+    # TODO This will need to be changed when UX is finalized
+    @integration_test
+    def test_subcommand_get_short_help_returns_help(self):
+        output = self.invoke_cli(self.cli_auth_params,
+                                 ['variable', 'get', '-h'], exit_code=0)
+        self.assertIn("usage: variable", output)
+
+    # TODO This will need to be changed when UX is finalized
+    @integration_test
+    def test_subcommand_get_long_help_returns_help(self):
+        output = self.invoke_cli(self.cli_auth_params,
+                                 ['variable', 'get', '--help'], exit_code=0)
+        self.assertIn("usage: variable", output)
+
+    @integration_test
+    def test_subcommand_get_help_returns_help(self):
+        with redirect_stderr(self.capture_stream):
+            self.invoke_cli(self.cli_auth_params,
+                            ['variable', 'get'], exit_code=1)
+        self.assertIn("Error the following arguments are required:", self.capture_stream.getvalue())
+
+    @integration_test
+    def test_subcommand_set_help_returns_help(self):
+        with redirect_stderr(self.capture_stream):
+            self.invoke_cli(self.cli_auth_params,
+                            ['variable', 'set'], exit_code=1)
+        self.assertIn("Error the following arguments are required:", self.capture_stream.getvalue())
+
+    # TODO This will need to be changed when UX is finalized
+    @integration_test
+    def test_subcommand_set_long_help_returns_help(self):
+        output = self.invoke_cli(self.cli_auth_params,
+                                 ['variable', 'set', '--help'], exit_code=0)
+        self.assertIn("usage: variable set", output)
+
+    # TODO This will need to be changed when UX is finalized
+    @integration_test
+    def test_subcommand_set_short_help_returns_help(self):
+        output = self.invoke_cli(self.cli_auth_params,
+                                 ['variable', 'set', '-h'], exit_code=0)
+        self.assertIn("usage: variable set", output)
+
+    # TODO This will need to be changed when UX is finalized
+    @integration_test
+    def test_subcommand_set_variable_with_values_returns_help(self):
```

Comment:
without value..

Reply:
great catch. Updating to `test_subcommand_set_variable_without_value_returns_help`